stratum/hal/lib/tdi/tdi_sde_wrapper.cc

```diff
@@ -115,6 +120,73 @@ TdiSdeWrapper* TdiSdeWrapper::GetSingleton() {
   return singleton_;
 }
 
+::util::StatusOr<uint32> TdiSdeWrapper::GetTableId(std::string &table_name) const {
+  const ::tdi::Table* table;
+  if (nullptr != tdi_info_) {
```

Comment:
Don't put the constant value before the variable name in a comparison. This is a misapplication of a terrible practice that cropped up in C in the 1980's. It reduces the readability of the code (and provides a bad example for other programmers) without adding any value.

Reply:
Is it terrible practice? I find that it avoids human error during coding where we may introduce an assignment instead of a comparison operator. 
Use cases such as programmer doing a ` if (x = 5)` instead of `if (x == 5)`

Having a constant on the lvalue avoids such mistake.

However, in terms of style to match rest of this codebase, we can change it back to `tdi_info_ != nullptr`

Reply:
Fixed